cmd/release/docs/main.go

```diff
@@ -126,7 +126,7 @@ func main() {
 	DeleteDocsDirectoryIfEmpty(&release)
 	log.Printf("Finished writing to %v doc(s)\n", len(docStatuses))
 
-	if openPR {
+	if *openPR {
 		err = OpenDocsPR(&release, docStatuses, *isBot)
```

Comment:
Intentional?

Reply:
yeah this was a bug. I had [fixed this issue with the other flags](https://github.com/aws/eks-distro/pull/493/files#diff-c11fdc4773cfc3c3984e5c0cb9093a6bd9d8fa375aa69c3d621ce45140a947ffL23-R65) awhile ago, but somehow this one slipped through